packages/node/test/tsconfig.json

```diff
@@ -1,7 +1,8 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
-    "downlevelIteration": true
+    "downlevelIteration": true,
+    "strict": false
```

Comment:
disabled strict mode in tests since it was due to some weird chai plugin shit and I imagine we're going to move this repo away from mocha/chai at some point.